configure.ac

```diff
@@ -1390,3 +1390,6 @@ m4_ifdef([project_ompi], [OMPI_CONFIG_FILES])
 m4_ifdef([project_oshmem], [OSHMEM_CONFIG_FILES])
 
 AC_OUTPUT
+
+OPAL_LOG_MSG([libnl : $opal_libnl_libs])
+OPAL_LOG_MSG([libnl3: $opal_libnl3_libs])
```

Comment:
Per the discussion on devel the other day about emitting a report at the end of configure (http://www.open-mpi.org/community/lists/devel/2016/01/18497.php), you might not want to add this to the *very* end of configure -- perhaps a few lines earlier, with the output of the final wrapper compiler flags...?